client/src/components/LabelImg/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./labelimg.css";
+
+const LabelImg = ({ ImgClassName, labelText, imgSrc }) => (
```

Comment:
There is many nested children inside this component. How is this going to look like?
@NoureldeanSaed

Reply:
When you're creating a component, you should expect the to be needing to change it's styles between once and a while.
So when you create a component `<Button ... />` for example, you've probably set it's `className` already, but what if I was your teammate and I wanted to give this button a margin of 5 more pixels? Or even change the color of the button component. I would have only one choice which is creating another button component only to give it 5 more pixels of padding for example.
You should think of the re-usability of your components while creating them.
Endpoint is: I understand your component has different tags using different attributes that are required by your `props`. But you should give the upmost parent (the `div`) the 're-usable` attributes in order to manipulate this component more easily, and I don't think that you need to give your image a className, because it is not the image which is representing the component. It is the parent div. Eventually it is your call. @amusameh